B-1B-set.xml

```diff
@@ -294,7 +294,7 @@ B-1B set file.
                 <menu n="10">
                     <label>B-1B</label>
                     <enabled type="bool">true</enabled>
-                    <item n="0">
+                    <!--item n="0">
```

Comment:
This button won't work in jsbsim anyway, so I'm fine with removing it. We should consider adding an autostart button that executes the startup checklist, though. That is very simple to implement.

Reply:
See here: https://wiki.flightgear.org/Aircraft_checklists#Automated_checklist_execution